editoast/src/client/redis_config.rs

```diff
@@ -0,0 +1,19 @@
+use clap::Args;
+use derivative::Derivative;
+
+#[derive(Args, Debug, Derivative, Clone)]
+#[derivative(Default)]
+pub struct RedisConfig {
+    #[derivative(Default(value = r#""localhost".into()"#))]
+    #[clap(long, env, default_value = "localhost")]
+    redis_host: String,
+    #[derivative(Default(value = "6379"))]
+    #[clap(long, env, default_value_t = 6379)]
+    redis_port: u16,
```

Comment:
You can add documentation to fields using `///`, this allows clap to add description in the CLI.
We don't do this everywhere but it is a good practice.

Reply:
Thanks I will do that :)